src/core/ledger/credAccounts.js

```diff
@@ -47,15 +47,19 @@ export function computeCredAccounts(
 ): CredAccountData {
   const grainAccounts = ledger.accounts();
   const userlikeInfo = new Map();
+  const intervals = credView.intervals();
+  const noIntervals = intervals.length === 0;
   for (const {address, credOverTime, description} of credView.userNodes()) {
-    if (credOverTime == null) {
+    if (noIntervals) {
+      userlikeInfo.set(address, {cred: [], description});
+    } else if (credOverTime == null) {
       throw new Error(
         `userlike ${NodeAddress.toString(address)} does not have detailed cred`
       );
+    } else {
+      userlikeInfo.set(address, {cred: credOverTime.cred || null, description});
```

Comment:
null case is untested, and for good reason. The downstream typing doesn't accept null, and the upstream credOverTime.cred attribute must be defined according to typing (which is why Flow doesn't fail this, because it knows  `|| null` will always be short circuited).

All that to say, you should be able to remove ` || null` and you probably oughta to avoid confusion.

Reply:
Good catch @blueridger ! This was dangling from an earlier iteration and I forgot to remove it.

Reply:
That was my guess :)